frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.ts

```diff
@@ -0,0 +1,47 @@
+type ParamsOptions = {
+  org?: string;
+  app?: string;
+};
+
+type PackagesRoute =
+  | 'dashboard'
+  | 'editorOverview'
+  | 'editorUiEditor'
+  | 'preview'
+  | 'editorPublish';
+
+const packagesRoutes = {
+  dashboard: '/dashboard',
+  editorOverview: '/editor/{{org}}/{{app}}/overview',
+  editorUiEditor: '/editor/{{org}}/{{app}}/ui-editor',
+  editorPublish: '/editor/{{org}}/{{app}}/deploy',
+  preview: '/preview/{{org}}/{{app}}',
+};
+
+export class PackagesRouter {
+  private app: string;
+  private org: string;
+
+  constructor(private paramsOptions: ParamsOptions) {
+    this.app = this.paramsOptions.app ?? '';
+    this.org = this.paramsOptions.org ?? '';
+  }
+
+  public navigateToPackage(packageRoute: PackagesRoute, subUrl?: string): void {
+    window.location.assign(`${this.getPackageNavigationUrl(packageRoute)}${subUrl ? subUrl : ''}`);
+  }
```

Comment:
We should rename the subUrl to be query-params to follow the standard naming convetions.

```suggestion
  public navigateToPackage(packageRoute: PackagesRoute, queryParams?: string = ''): void {
    window.location.assign(`${this.getPackageNavigationUrl(packageRoute)}${queryParams}`);
  }
```

Reply:
I agree, but I would still need the check in the method because of this: 
<img width="411" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/256d3469-1aed-4ecd-bacd-2a118151396e">

Reply:
Looks like you resolved it by removing the default value? 😊

Reply:
Correct 😄